src/scripts/main.js

```diff
@@ -1,3 +1,110 @@
 'use strict';
 
-// write code here
+class Table {
+  TBODY = document.querySelector('.field tbody');
+  APPEND_ROW_BUTTON = document.querySelector('.append-row');
+  REMOVE_ROW_BUTTON = document.querySelector('.remove-row');
+  APPEND_COLUMN_BUTTON = document.querySelector('.append-column');
+  REMOVE_COLUMN_BUTTON = document.querySelector('.remove-column');
+
+  rowSize = 4;
+  cellsSize = 4;
+
+  constructor() {
+    this.getTableSize();
+    this.addListeners();
+  }
+
+  getTableSize() {
+    this.ROWS = this.TBODY.querySelectorAll('tr');
+
+    if (this.ROWS.length > 0) {
+      const cells = this.ROWS[0].cells;
+
+      this.rowSize = this.ROWS.length;
+      this.cellsSize = cells.length;
+    }
+  }
+
+  createRow = () => {
+    // Next line for test, disabled button works correctly
+    if (this.rowSize === 10) {
```

Comment:
Avoid magic numbers in your code, it's better to create a constant for the maximum number of rows and columns (10 in this case), and use that constant instead. This makes your code easier to read and maintain.